### PR TITLE
Feature: Add read acknowledgment to channel sdk doc. RD-22073

### PR DESCRIPTION
**Pull request by brenauj**

### CODE REVIEW

File: docs/sdks/source-sdk/action-details.md [hunk 1]
```diff
@@ -55,7 +55,8 @@ We'll always retry to send the message after a message level error. If you want
     "objects": {
         "messages": ["create", "show", ...],
         "private_messages": [...],
-        "threads": [...]
+        "threads": [...],
+        "status": ["mark_as_read"]
```
Comment (brenauj) [inline docs/sdks/source-sdk/action-details.md:59]: Screenshot from 2022-06-08 14-59-50

File: docs/sdks/source-sdk/objects.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
 # Intro to SDK Objects
 
-Currently the SDK defines 4 types of objects:
+Currently the SDK defines 5 types of objects:
 
 * Messages: publicly available messages exchanged between any number of users. Examples: blog comments, forum posts.
 * PrivateMessages: messages between 2 or more users that cannot be seen by others. Examples: direct messages on Twitter.
 * Threads: provide a way to structure the messages. Examples: blog posts, forum threads. Not mandatory (eg. Twitter).
 * Users: every other object type must have an author. This is the only object type that has no actions available.
+* Status: provides a way to update messages and private messages, like marking them as read. Used only in [Send API](../send-api)
```
Comment (brenauj) [inline docs/sdks/source-sdk/objects.md:9]: Screenshot from 2022-06-08 14-57-41

File: docs/sdks/source-sdk/objects.md [hunk 3]
```diff
@@ -138,12 +139,20 @@ We ignore extra fields and you only need to provide fields marked as required. A
...
\ No newline at end of file
+| context_data | Hash | Data specific to the User, that will be stored in Engage Digital at the identity and identitygroup level. Expect a hash having the (expected) form: `{'field_name' => 'value'}`. *The keys must be [a-zA-Z-]*. Example: {`"country": "Germany", "age": "34", "fruits": ["Apple", "Pear"]}`. |
+
+## Status
+
+We ignore extra fields and you only need to provide fields marked as required. Used only in [Send API](../send-api)
+
+| Name | Type | Description |
+|-|-|-|
+| id | String | ***Required***, ID of the message or private message on the third party system (yours). This is the ID you return on `message.create` and `private_message.create` actions |
```
Comment (brenauj) [inline docs/sdks/source-sdk/objects.md:158]: Screenshot from 2022-06-08 14-57-59

File: docs/sdks/source-sdk/send-api.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # Introduction to Send API
 
-The Send API allows you to send contents instantly to Engage Digital. It's purpose is to build a near realtime API when the third party to be connected can support it. This API is optional and is a way to mitigate the slow import (~2 min) of the [polling approach](../polling).
+The Send API allows you to send contents instantly to Engage Digital. Its purpose is to build a near realtime API when the third party to be connected can support it. This API is optional and is a way to mitigate the slow import (~2 min) of the [polling approach](../polling).
+
+It also allows you to mark agent contents as read, which is visible by agents when '[view.messaging](../options/#viewmessaging)' option is enabled.
```
Comment (brenauj) [inline docs/sdks/source-sdk/send-api.md:5]: Screenshot from 2022-06-08 14-56-31

File: docs/sdks/source-sdk/send-api.md [hunk 5]
```diff
@@ -52,6 +54,7 @@ This section lists the parameters you must send for all actions and specifies wh
 | Private messages | create |
 | Threads | create |
 | Users | create |
+| Status | mark_as_read |
```
Comment (brenauj) [inline docs/sdks/source-sdk/send-api.md:57]: Screenshot from 2022-06-08 14-56-50

File: docs/sdks/source-sdk/send-api.md [hunk 6]
```diff
@@ -79,6 +82,25 @@ This section lists the parameters you must send for all actions and specifies wh
...
+  }
+}
+```
+
+#### Response
+
+``` json
+ok
+```
+
```
Comment (brenauj) [inline docs/sdks/source-sdk/send-api.md:103]: Screenshot from 2022-06-08 14-57-08

**Review by brenauj — COMMENTED** [review]
added screenshots

File: docs/sdks/source-sdk/objects.md (hunk 2 above)
Comment (DrissTM) [inline docs/sdks/source-sdk/objects.md:9]:
```suggestion
* Status: provides a way to update messages and private messages, like marking them as read. Only used in [Send API](../send-api)
```

**Comment by DrissTM** [comment]
one minor suggestion, otherwise LGTM :+1:

**Pull request merged**